app/views/layouts/application.html.erb

```diff
@@ -1,17 +1,17 @@
 <!DOCTYPE html>
 <html>
+  <body class="site">
```

Comment:
I'm referencing this [https://philipwalton.github.io/solved-by-flexbox/demos/sticky-footer/](url) to keep the footer pinned to the bottom...